### ISSUE TITLE
connection error?

### ISSUE THREAD

**Issue by bazancik** [Ascoware/get-iplayer-automator#52]
Hi, 
I've been using GiP for a while and all of sudden all stopped working: it starts up fine but the cache update does not seem to be working. every search returns 'no shows' message. download from the current www also fails: adds the programme to the list, when try to download 'Unknown: PID Not Found' message comes up.

When tried to report the issue from within the app i get: There was a problem sending your feedback.  Please try again at another time

Not sure what the issue might be and of it applies to more folks out there?
No DNS / VPN on the machine, no firewall, no proxy. 

Cheers for any pointers / help as cannot find the issue.

**Comment by DangerMouse43** [comment]
Hi,

I have had almost the same issue. I managed to find some programmes but I can see on the iPlayer website that there are specific programmes available but when I type them in the search nothing is returned and I just get the popup 'No Shows Found'. The app did a load of re-indexing on startup and I could find more after that but still most programmes are missing.

**Comment by MacInWin** [comment]
ITV searching has been broken by a change ITV made. To download ITV, open your browser, go to ITV and find the episode you want, then tell GiP to use the current website. Clumsy, but works. I don't think there is going to be a fix for searching as the change was on ITV's side and from what I know, pretty much killed it.

**Comment by DangerMouse43** [comment]
My issue was on the BBC iPlayer content not appearing in the search not ITV. How do I tell GiA to use the current site? Will that work for the BBC too?

**Comment by MacInWin** [comment]
Yes, it will. Get to the exact show and episode in your browser then tell GiA to Use Current Webpage and off you go.

**Comment by Lordsnooty05** [comment]
Just remember when you use GIA and load a show via "Use Current Webpage" you MUST use SAFARI Browser GIA only works with safari. Plus when you search for an ITV Program you have to use ITV Hub at https://www.itv.com/hub/itv

**Comment by WilliamOfGaunt** [comment]
Point of information: GiA also works with Chrome (can be selected as default browser).

**Comment by Lordsnooty05** [comment]
WilliamofGaunt I didn't know that....but hey I don't like Chrome that much....more of a Firefox and Opera Lover....but good to know there is an alternative to Safari that works with GIA....Thanks
